catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -145,11 +148,10 @@ protected Object executeWithSubject() throws Exception {
             return null;
         }
 
-        if (!DEFAULT_TRANSFORMER_ID.matches(transformerId)) {
+        if (!FALLBACK_TRANSFORMER_ID.matches(transformerId)) {
             transformers = getTransformers();
             if (transformers == null) {
-                console.println(transformerId + " is an invalid metacard transformer.");
-                return null;
+                console.println(transformerId + " is an invalid metacard transformer.  Defaulting to Java Serialized Object.");
```

Comment:
I don't think we want to default to any transformer if the transformer provided is not valid.  Rather the command should fail with a clear explanation (perhaps including the list of valid transformer ids).